src/ProofOfConcepts/CommunicationError/CommunicationErrorStatusInitial.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import {
+  Alert,
+  AlertActionCloseButton,
+  Button,
+  ClipboardCopy,
+} from '@patternfly/react-core';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
+
+import { useTranslation } from '@rhoas/app-services-ui-components';
+
+import './CommunicationErrorStatus.css';
+
+export const ExpandableAlerts = () => {
+  const { t } = useTranslation('cos-ui');
+
+  return (
+    <React.Fragment>
+      <Alert
+        isExpandable
+        isInline
+        variant="danger"
+        title={t('communicationErrorTitle')}
+        actionClose={
+          <AlertActionCloseButton
+            onClose={() => alert('Clicked the close button')}
+          />
+        }
+        actionLinks={<React.Fragment></React.Fragment>}
```

Comment:
I think you can also remove this prop. We don't have action links for this alert.